executable/persisted/tests/Test.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const path = require("node:path");
+
+const {
+  deployAndRun,
+  authTypes,
+  getTestDescription,
+} = require("../../../tests/gqltest.js");
+
```

Comment:
Is there any way to define the REPO_ROOT like we have done in our Makefiles?  this `../../../..` nonsense always catches me.  somehow I could do this *once* at the top of the file more easily -- I guess in this case it's only used once...